### PR TITLE
#420 - updated phpoffice package

### LINKED ISSUE #420: Update phpoffice/phpword
We should update phpoffice/phpword to the latest version (1.2 at this moment 3.04.2024) and check if everything works as expected.

Notes:
> It doesn't work as expected. For some reason new version of package creates files in /tmp with some random names, but doesn't remember these names and returns something different (and non-existing).

### PR DESCRIPTION
**Pull request by kamilpiech97**
Should close #420.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "keywords": ["toby", "laravel", "hr"],
     "license": "MIT",
     "require": {
-        "php": "^8.3.8",
+        "php": "^8.3.0",
```
Comment (mtracz) [inline composer.json:8]: What is the reason of this change? PHP in the docker image is 8.3.8, and any composer install/update command will failed in this case.

File: app/Domain/ResumeGenerator.php [hunk 2]
```diff
@@ -24,7 +24,10 @@ public function generate(Resume $resume): string
         $this->fillEducation($processor, $resume);
         $this->fillProjects($processor, $resume);
 
-        return $processor->save();
+        $path = storage_path("app/resumes/" . now() . ".docx");
```
Comment (mtracz) [inline app/Domain/ResumeGenerator.php:27]: Which date format will be used here? How path will looks like?

Comment (kamilpiech97) [reply app/Domain/ResumeGenerator.php:27]: its temp file, we are deleting them after download

File: composer.json (hunk 1 above)
Comment (kamilpiech97) [reply composer.json]: fixed, I had failure earlier because I do not have the newest build of the app.. and I made change to 8.3.0 version.

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by mtracz — APPROVED** [review]
